File: CS-Examples/07_Conversion/HtmlToExcel.cs
using System;
using System.Windows.Forms;
using Spire.Xls;


namespace HtmlToExcel
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        private void btnRun_Click(object sender, System.EventArgs e)
        {
            //Html path
            string filePath = @"..\..\..\..\..\..\Data\HtmlToExcel.html";

            //Create a workbook
            Workbook workbook = new Workbook();

            //Load html
            workbook.LoadFromHtml(filePath);

            //Save to Excel file
            string result = "HtmlToExcel_result.xlsx";
            workbook.SaveToFile(result, ExcelVersion.Version2013);

            // Dispose of the workbook object to release resources
            workbook.Dispose();

            //Launch the file
            ExcelDocViewer(result);
        }
        private void ExcelDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }

    }
}

File: CS-Examples/07_Conversion/HtmlToExcel.vb
Imports System.Collections
Imports System.ComponentModel

Imports Spire.Xls
Imports System.Text
Imports System.IO

Namespace HtmlToExcel
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Specify the file path of the HTML file to be loaded
            Dim filePath As String = "..\..\..\..\..\..\Data\HtmlToExcel.html"

            ' Create a new instance of Workbook
            Dim workbook As New Workbook()

            ' Load the HTML file into the workbook
            workbook.LoadFromHtml(filePath)

            ' Specify the output file name for saving as Excel file
            Dim result As String = "HtmlToExcel_result.xlsx"

            ' Save the workbook to an Excel file in Excel 2013 format
            workbook.SaveToFile(result, ExcelVersion.Version2013)
            ' Release the resources used by the workbook
            workbook.Dispose()

            'Launch the file
            ExcelDocViewer(result)
		End Sub
		Private Sub ExcelDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub

	End Class
End Namespace
